docs/index.rst

```diff
@@ -4120,6 +4119,52 @@ argument with code ``0`` to use ``last prediction`` and ``1`` for
     local_model.predict({"petal length": 3, "petal width": 1},
                         missing_strategy=PROPORTIONAL)
 
+If you would like a per-class prediction of the confidence or
```

Comment:
Maybe something like:

For classification models, it may be useful sometimes to get the predicted probabilites (or confidences) associated to every possible category in our objective field. In this case, you can use the ...